resdk/resources/collection.py

```diff
@@ -163,6 +163,23 @@ def download(self, file_name=None, field_name=None, download_dir=None):
 
         self.resolwe._download_files(files, download_dir)  # pylint: disable=protected-access
 
+    def delete(self, force=False, delete_content=False):
+        """Delete the resource object from the server."""
```

Comment:
Mention `delete_content` in the docstring. It would be also nice to describe `force` parameter, but I know it is also not documented in the base class.